FCWTAPI/CWTObject.cs

```diff
@@ -230,10 +235,10 @@ public void CalculateTimeAxis()
             {
                 throw new ArgumentNullException("OutputCWT", "Output CWT must be calculated prior to calculating a time axis for it");
             }
-            double [] timeArray = new double[OutputCWT.GetLength(1)];
+            double [] timeArray = new double[OutputCWT.RealArray.GetLength(1)];
             double timeStep = 1 / (double)SamplingRate;
             double currentTime = 0;
-            for (int i = 0; i < OutputCWT.GetLength(1); i++)
+            for (int i = 0; i < OutputCWT.RealArray.GetLength(1); i++)
```

Comment:
I suspect that this may be the inefficiency since this for loop is going to require that OutputCWT.RealArray.GetLength(1) will have to be calculated for every i, and given typical data sizes this will lead to 10's to hundreds of thousands of unnecessary calculations.